draft-ietf-masque-connect-udp-listen.md

```diff
@@ -251,7 +343,7 @@ back to the client.
 
             <--------  DATAGRAM
                          Quarter Stream ID = 11
-                         Context ID = 2
+                         Context ID = 0
```

Comment:
I don't think that works — context ID 0 is reserved for the standard UDP datagram format (so no address+port)

Reply:
Ah I see, We could use any other number I guess. I can swap it out for 2

Reply:
FWIW this is an extension to connect-udp so it wouldn't be crazy to reuse context ID 0, because in this case datagrams with context ID zero have no logical semantics. Both options make sense

Reply:
Interesting, wasn't clear that connect-udp would allow that — I assumed that's why the draft originally used 2 there.

```
The Context ID value of 0 is reserved for UDP payloads, while non-zero values are dynamically allocated. Non-zero even-numbered Context IDs are client-allocated, and odd-numbered Context IDs are proxy-allocated.
```

Reply:
Everything's allowed if the extension is negotiated. I guess the nice property of using 2 instead of 0 is that the client can send datagrams before receiving the response headers